bindings/java/local.mk

```diff
@@ -67,10 +71,20 @@ bindings/java/foundationdb-client.jar: bindings/java/.classstamp lib/libfdb_java
 	@cp lib/libfdb_java.$(DLEXT) bindings/java/classes/main/lib/$(PLATFORM)/$(java_ARCH)/libfdb_java.$(java_DLEXT)
 	@jar cf $@ -C bindings/java/classes/main com/apple/foundationdb -C bindings/java/classes/main lib
 
+bindings/java/foundationdb-junit-tests.jar: bindings/java/.classstamp bindings/java/.dependencies/hamcrest-all-1.3.jar
+	@echo "Building       $@"
+	@jar cf $@ -C bindings/java/classes/junit com/apple/foundationdb
+
+bindings/java/.dependencies/hamcrest-all-1.3.jar:
+	@echo "Download       hamcrest-all"
+	@mkdir -p bindings/java/.dependencies
+	@curl -so bindings/java/.dependencies/hamcrest-all-1.3.jar "http://search.maven.org/remotecontent?filepath=org/hamcrest/hamcrest-all/1.3/hamcrest-all-1.3.jar"
```

Comment:
I can push .dependencies folder (with junit and hamcrest) into git and remove download parts from local.mk.
What do you think about this solution?

Reply:
Hm, I don't really like that solution, though I see how it would remove the download problem. Given how infrequently we run the unit tests (and how little coverage they provide), I don't know if it's worth crossing the Rubicon of checking jars in to the repo just for this. But I can see what other people think.